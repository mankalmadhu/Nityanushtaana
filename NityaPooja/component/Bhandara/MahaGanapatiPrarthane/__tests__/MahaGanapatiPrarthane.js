import 'react-native';
import React from 'react';
import ಮಹಾಗಣಪತಿಪ್ರಾರ್ಥನೆ from '../MahaGanapatiPrarthane';

import Renderer from 'react-test-renderer';

jest.mock('../../../Avarana/Avarana',()=>'ಆವರಣ');
jest.mock('../../../Vishaya/Sheershike',()=>'ಶೀರ್ಷಿಕೆ');
jest.mock('../../../Vishaya/Purvanga',()=>'ಪೂರ್ವೋತ್ತರ');
jest.mock('../../../Vishaya/Viniyoga',()=>'ವಿನಿಯೋಗ');
jest.mock('../../../Vishaya/Vidhana',()=>'ವಿಧಾನ');

test("ಮಹಾಗಣಪತಿಪ್ರಾರ್ಥನೆ" , () => {

expect(Renderer.create(
    <ಮಹಾಗಣಪತಿಪ್ರಾರ್ಥನೆ />
).toJSON()).toMatchSnapshot();

});