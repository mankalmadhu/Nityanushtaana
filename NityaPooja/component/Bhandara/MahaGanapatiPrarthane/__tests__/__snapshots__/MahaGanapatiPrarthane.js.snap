// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ಮಹಾಗಣಪತಿಪ್ರಾರ್ಥನೆ 1`] = `
<ಆವರಣ>
  <ಶೀರ್ಷಿಕೆ>
    ಮಹಾಗಣಪತಿ ಪ್ರಾರ್ಥನೆ
  </ಶೀರ್ಷಿಕೆ>
  <ಪೂರ್ವೋತ್ತರ>
    ಆದೌ ನಿರ್ವಿಘ್ನತಾ ಸಿದ್ಧ್ಯರ್ಥಂ ಶ್ರೀ ಮಹಾಗಣಪತಿ ಪ್ರಾರ್ಥನಾ೦ ಕರಿಷ್ಯೇ ॥
  </ಪೂರ್ವೋತ್ತರ>
  <ವಿನಿಯೋಗ>
    ಗ॒ಣಾನಾ᳚೦ ತ್ವಾ ಗ॒ಣಪ॑ತಿಗ್೦ ಹವಾಮಹೇ ಕ॒ವಿ೦  ಕ॑ವೀ॒ನಾಮು॑ಪ॒ಮಶ್ರ॑ವಸ್ತಮ೦ । ಜ್ಯೇ॒ಷ್ಠ॒ರಾಜ॒೦ ಬ್ರಹ್ಮ॑ಣಾ೦ ಬ್ರಹ್ಮಣಸ್ಪತ॒ ಆ ನಃ॑ ಶೃ॑ಣ್ವನ್ನೂ॒ತಿಭಿ॑ಸ್ಸೀದ॒ ಸಾದ॑ನ೦ ॥
  </ವಿನಿಯೋಗ>
  <ವಿನಿಯೋಗ>
    ವಕ್ರತುಂಡ ಮಹಾಕಾಯ ಕೋಟಿ ಸೂರ್ಯ ಸಮಪ್ರಭ । ನಿರ್ವಿಘ್ನಂ ಕುರು ಮೇ ದೇವ ಸರ್ವ ಕಾರ್ಯೇಷು ಸರ್ವದಾ ॥
  </ವಿನಿಯೋಗ>
  <ವಿನಿಯೋಗ>
    ಓಂ ಸುಮುಖಾಯ ನಮಃ । ಏಕದಂತಾಯ ನಮಃ । ಕಪಿಲಾಯ ನಮಃ । ಗಜಕರ್ಣಕಾಯ ನಮಃ ॥ ಲ೦ಬೋದರಾಯ ನಮಃ । ವಿಕಟಾಯ ನಮಃ । ವಿಘ್ನರಾಜಾಯ ನಮಃ । ಗಣಾಧಿಪಾಯ ನಮಃ ॥ ಧೂಮ್ರಕೇತವೇ ನಮಃ । ಗಣಾಧ್ಯಕ್ಷಾಯ ನಮಃ । ಫಾಲಚ೦ದ್ರಾಯ ನಮಃ । ಗಜಾನನಾಯ ನಮಃ ॥
  </ವಿನಿಯೋಗ>
  <ಪೂರ್ವೋತ್ತರ>
    ಇತಿ ಶ್ರೀ ಮಹಾಗಣಪತಿ ಪ್ರಾರ್ಥನಾ೦ ಸಮರ್ಪಯಾಮಿ ॥
  </ಪೂರ್ವೋತ್ತರ>
  <ವಿಧಾನ>
    (ಮಹಾಗಣಪತಿಗೆ ಗ೦ಧ ಪುಷ್ಪಗಳನ್ನು ಅರ್ಪಿಸಿ ಪ್ರಾರ್ಥನೆ ಮಾಡಿಕೊಳ್ಳುವುದು.)
  </ವಿಧಾನ>
</ಆವರಣ>
`;
